config.toml

```diff
@@ -4,6 +4,7 @@ allowed-mailing-lists-domains = [
     "docs.rs",
     "rust-embedded.org",
     "rustconf.com",
+    "teams.rust-embedded.org",
```

Comment:
I think we can add this later when/if we actually move mailing lists over.

Reply:
i can remove it if you prefer, but my intent is to move the mailing lists over asap (though i am not sure about the rest of this process?)